src/components/astro/AstroClassroomsTable.jsx

```diff
@@ -21,8 +21,15 @@ import {
   CLASSROOMS_INITIAL_STATE, CLASSROOMS_PROPTYPES
 } from '../../ducks/classrooms';
 
-const AstroClassroomsTable = (props) => {
-  // TODO: Replace classroom.zooniverse_user_group_id with actual serialized user group id when available
+function calculateCompleteness(assignment, classroom) {
+  const classificationsTarget = +assignment.metadata.classifications_target;
```

Comment:
Two questions:
1. Is there any chance that `classificationsTarget` can ever be 0?
2. I assume the `+` is a method of converting what could possibly be a string into a number, correct?